src/background.js

```diff
@@ -1,9 +1,20 @@
 let tabHistory = {};
 let currentTabId;
-let currentTabJustRemoved = false;
+let currentTabJustRemoved = false; // No-use if `defaultTabClosingBehavior`.
+let defaultTabClosingBehavior = false;
 
 function init() {
 	tabHistory[-1] = {id: null}; // Dummy start.
+	chrome.contextMenus.create({
+		type: "checkbox",
+		title: "Go to last tab after closing current one",
```

Comment:
I think we should make this as explicit and clear as possible: 

"Go to previously viewed tab after closing current one"

Do you agree?

Reply:
Great!